test/system/280-update.bats

```diff
@@ -161,4 +161,552 @@ device-write-iops   = /dev/zero:4000 | - | -
     run_podman rm -f -t0 testctr
 }
 
+function _create_container_check_change_of_HealthCheck_configuration_with_podman_update(){
+    local ctrname="$1"
+    local msg="$2"
+    local format="$3"
+    local flag="$4"
+    local value="$5"
+    local expect="$6"
+    local expect_msg="$7"
```

Comment:
Third, this is a completely unnecessary argument. It is identical to `$format` except in the `health-on-failure` test where I have no idea if that's a dup-typo or intentional. Please eliminate this.